### ISSUE TITLE
Enhancment: use same background for masked pixels between plot_band and plot_multiple_bands

### ISSUE THREAD

**Issue by atoparseks** [EOA-team/eodal#56]
When I use scene.plot_multiple_bands, masked pixels are black:
![image](https://github.com/EOA-team/eodal/assets/3286454/f9c871c7-ebcd-4e79-a722-5dfd6a6b5e2b)

For one band I can use scene.plot_band , and the masked pixels are white

I suggest white for both cases. Or black. This doesn't look good when placed side to side.
![image](https://github.com/EOA-team/eodal/assets/3286454/ea983de9-eb00-423f-925f-4431205d59b9)

**Comment by lukasValentin** [comment]
Hi @atoparseks, I now figured out how to change the color of the nodata pixels, i.e., those pixels masked out. To do so, I added a **new optional keyword argument** in `RasterCollection.plot_multiple_bands` named `nodata_color`. By default it is set to 'white' to mimic the default behavior of the `Band.plot` method. 

`nodata_color` can be set, however, to any color name available in matplotlib (see [here](https://matplotlib.org/stable/gallery/color/named_colors.html) for a full list of named colors).

The code snippet below shows the usage:

```python
from eodal.core.band import Band, GeoInfo
from eodal.core.raster import RasterCollection

import numpy as np

# open sample data
rc = RasterCollection.from_multi_band_raster(fpath_raster='data/20190530_T32TMT_MSIL2A_S2A_pixel_division_10m.tiff')
# mask some pixels so we can test the new functionality
masked = rc.mask(mask=rc['B8A'] > 1000, bands_to_mask=rc.band_names)

# no-data colored white [255,255,255] (new default) -> same no-data color as Band.plot
masked.plot_multiple_bands(band_selection=['B02', 'B03', 'B04'], nodata_color='white')

# i.e., omitting the nodata_color argument will have the same effect as the
# previous statement
masked.plot_multiple_bands(band_selection=['B02', 'B03', 'B04'])

# no-data colored black [0,0,0] (old default until version 0.2.0)
masked.plot_multiple_bands(band_selection=['B02', 'B03', 'B04'], nodata_color='black')

# no-data colored in an example matplotlib color, e.g. "red"
masked.plot_multiple_bands(band_selection=['B02', 'B03', 'B04'], nodata_color='red')

# or some other color, e.g., "darkkhaki"
masked.plot_multiple_bands(band_selection=['B02', 'B03', 'B04'], nodata_color='darkkhaki')
```

**Comment by lukasValentin** [comment]
I close this issue since it is resolved by #61

**Comment by atoparseks** [comment]
Thanks! Looks really neat!
